src/snowflake/snowpark/_internal/utils.py

```diff
@@ -519,12 +519,12 @@ def result_set_to_rows(
         col_names = [col.name for col in result_meta]
         rows = []
         for data in result_set:
-            row = Row(*data)
+            row = Row() if data is None else Row(*data)
```

Comment:
I'm thinking we should gracefully alert users when data is `None` so they know the reason of the error and how to avoid it.

Reply:
logger.warning? However, we don't know the exact reason why it can be None (as we chatted yesterday, there might be multiple reasons).

Reply:
We may need to update python connector. Does connector have more information to tell the root cause?

Reply:
not yet, see here https://github.com/snowflakedb/snowpark-python/issues/496#issuecomment-1284470345